### ISSUE TITLE
Will you add other algorithms like ConvE/ConvR ?

### ISSUE THREAD

**Issue by EdwardLorenz** [facebookresearch/PyTorch-BigGraph#109]

**Comment by lerks** [comment]
Something like this has been discussed in #52. Our approach for now has been to allow PBG to be extended so that users can implement and try out other models. We have a plugin system that allows you to register custom operators, comparators and loss functions. So far this has proven to be enough to allow most other models (#52 is in fact about overcoming some limitations of that). If the models you're trying to implement cannot fit our current abstractions, then we may consider loosening them up and extend them, but we'd need to first be convinced that these models do provide some benefit, i.e., that they work better than the currently-supported models in at least some scenarios.

**Comment by EdwardLorenz** [comment]
thanks a lot
